src/main/resources/db/migration/V35__audit_report.sql

```diff
@@ -0,0 +1,23 @@
+drop table if exists reported_adjudication_status_audit;
+
+create table reported_adjudication_status_audit
+(
+    id                             serial primary key,
+    reported_adjudication_fk_id    bigint references reported_adjudications (id),
+    status                         varchar(32)  not null,
+    status_reason                  varchar(128),
+    status_details                 varchar(4000),
+    create_user_id                 varchar(32)   not null,
+    create_datetime                timestamp     not null,
+    modify_user_id                 varchar(32),
```

Comment:
There isn't actually a reason for these `modify` cols, but may be easiest to leave them in